src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -120,12 +112,13 @@ static bool wall_tty_match(const char *path, bool is_local, void *userdata) {
          * advantage that the block will automatically go away if the
          * process dies. */
 
+        _cleanup_free_ char *p = NULL;
         if (asprintf(&p, "/run/systemd/ask-password-block/%u:%u", major(st.st_rdev), minor(st.st_rdev)) < 0) {
-                log_oom();
+                log_oom_debug();
                 return true;
         }
 
-        fd = open(p, O_WRONLY|O_CLOEXEC|O_NONBLOCK|O_NOCTTY);
+        _cleanup_close_ int fd = open(p, O_WRONLY|O_CLOEXEC|O_NONBLOCK|O_NOCTTY);
         if (fd < 0) {
                 log_debug_errno(errno, "Failed to open the wall pipe: %m");
```

Comment:
Hmm, this log msg should be more specific that this is what we're expecting? Currently it is pretty misleading to report this as an error? (also in wall_tty_block())